lib/shared/drive-constraints.js

```diff
@@ -231,12 +231,10 @@ exports.isDriveDisabled = (drive) => {
  * }
  */
 exports.isDriveValid = (drive, image) => {
-  return _.every([
-    !this.isDriveLocked(drive),
-    this.isDriveLargeEnough(drive, image),
-    !this.isSourceDrive(drive, image),
+  return !this.isDriveLocked(drive) &&
```

Comment:
?

Reply:
Oh, that was due to testing, but it does the same – this way faster & more efficient though, as it doesn't waste-allocate an array on every call.

Reply:
...as long as the linter doesn't complain at you for not using lodash ;-)

Reply:
Nah, linter is not complaining (thankfully)